doc/installation.rst

```diff
@@ -9,43 +9,45 @@ Installation
    :local:
    :backlinks: top
 
-Platforms
----------
+Supported Platforms
+-------------------
+
+*SfePy* is known to work on various flavors of Linux, Intel-based MacOS and
+Windows. It can work with both Python 2.7 and Python 3.x.
 
-*SfePy* is known to work on various flavors of Linux, on Intel Macs and
-Windows. It can work with both Python 2.7 and Python 3. Depending on Python
-installation and OS used, replacing ``python`` by ``python3`` might be required
-in all the commands below (e.g. in :ref:`compilation`) in order to use Python 3.
+Depending on Python installation and OS used, replacing ``python`` by ``python3`` might be required
+in all the commands below (e.g. in :ref:`compilation`) in order to use Python 3. However we strictly recommend to use *Python Virtual Environments* (see `Python virtualenv`_ or `Conda Environments`_) for these purposes.
 
-The easiest way to install SfePy is to use `Anaconda`_, see `Multi-platform
-Distributions Notes`_.
+The far easiest way we recommend to use *SfePy* on any supported platform is to install `Anaconda`_ Python distribution
+(see `Multi-platform Python Distributions Notes`_) and follow `Generic SfePy Installation Instructions`_.
```

Comment:
I would just use:

The easiest way in install *SfePy* and its dependencies on any supported platform is to install `Anaconda`_ Python distribution, see `Multi-platform Python Distributions Notes`_.

The `Generic SfePy Installation Instructions`_ are not needed, when installing sfepy from conda-forge.